CONTRIBUTING.md

```diff
@@ -37,3 +37,30 @@ Contributions made by corporations are covered by a different agreement than the
  
 3) Add or change sample Gist IDs to the [index file](https://github.com/dart-lang/dart-pad/blob/master/web/index.html#L54),
    and submit a PR for review.
+
+## How to run DartPad locally
+
+* To run the DartPad against the regular serving backend:  `grind serve`.  This serves the
+DartPad frontend locally on port 8000.
+
+* To run DartPad against a local version of the dart-services backend:
+```bash
+pub global activate grinder
```

Comment:
This should move above – since the first bullet mention using `grind serve`

Reply:
Also can mention that it runs fine with `pub run build_runner serve` – and reloads a lot faster on updates, too!

Reply:
@kevmoo It's great that that works, but given that there are a lot of hand-hacked bits in the grinder script for how to build this thing, I'm reluctant to recommend that method.  grind serve is guaranteed to get you what would be deployed.

Reply:
done